ArraySimulation/MPPT/GlobalMPPTAlgo/GlobalAlgorithm.py

```diff
@@ -0,0 +1,127 @@
+"""
+GlobalAlgorithm.py
+
+Author: Afnan Mir, Array Lead (2021).
+Contact: afnanmir@utexas.edu
+Created: 02/06/2021
+Last Modified: 02/06/2021
+
+Description: The Global MPPTAlgorithm (Maximum Power Point Tracker) class is a concrete
+base class that provides a common API for derived classes to use. The
+GlobalAlgorithm class enables users to calculate or predict voltage setpoints that
+would maximize the output power of the PVSource given a set of input conditions.
+"""
+from ArraySimulation.MPPT.MPPTAlgorithms.MPPTAlgorithm import MPPTAlgorithm
+from ArraySimulation.MPPT.MPPTAlgorithms.PandO import PandO
+from ArraySimulation.MPPT.MPPTAlgorithms.IC import IC
+from ArraySimulation.MPPT.MPPTAlgorithms.FC import FC
+from ArraySimulation.MPPT.MPPTAlgorithms.Ternary import Ternary
+from ArraySimulation.MPPT.MPPTAlgorithms.Golden import Golden
+from ArraySimulation.MPPT.MPPTAlgorithms.Bisection import Bisection
+
+class GlobalAlgorithm:
+    MAX_VOLTAGE = 100
+
+    MAX_VOLTAGE_PER_CELL = 0.8
+
+
+    def __init__(self, numCells = 1, MpptAlgoType = "Default", strideType = "Fixed"):
```

Comment:
I suggest changing MpptAlgoType to MPPTAlgoType, since MPPT is an acronym.

Reply:
In fact, I suggest changing MPPTAlgoType to MPPTLocalAlgoType for clarity.

Reply:
Additionally, I'd like to see an extra parameter MPPTGlobalAlgoType defined here.